values/velero/velero.gotmpl

```diff
@@ -19,9 +19,11 @@ metrics:
 snapshotsEnabled: false
 {{- end }}
 
-upgradeCRDs: false
+upgradeCRDs: true
```

Comment:
Do we need this?

Reply:
I tested it and on this version I did not encounter any issues. I'd prefer to keep this when/if it works

Reply:
I am a bit concerned that for old Velero version the upgrade CRDs suddenly stopped working. If the upgradeCRDs only performs the `kubectl` command then we could do it via the upgrade scripts. I leave this decision to you.

Reply:
Updated to use the kubectl installation